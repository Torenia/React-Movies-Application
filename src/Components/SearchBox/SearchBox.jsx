import styled from 'styled-components';

export const SearchBox = styled.div`
  display: flex;
  justify-content: flex-start;
  align-items: center;
  margin-left: 110px;
    
  button {
    color: #fff;
    background-color: #f65261;
    font-size: 15px;
    text-transform: uppercase;
    border: none;
    border-radius: 5%;
    outline:none;
    cursor: pointer;
    padding: 10px 20px;
    margin: 0;
    width: 160px;
  }
    
  input {
    width: 415px;
    height: 35px;
    color: #fff;
    margin-right: 15px;
    padding-left: 20px;
    background-color: #232323;
    border: none;
    border-radius: 5%;
    opacity: 0.5;
    outline:none;
      
    &::placeholder {
      font-size: 15px;
    }
  }
`;
