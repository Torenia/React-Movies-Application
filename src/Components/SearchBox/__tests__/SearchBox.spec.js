import React from 'react';

import { SearchBox } from '../SearchBox';

describe('<SearchBox />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SearchBox />);
  });

  it('should render a search box', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
