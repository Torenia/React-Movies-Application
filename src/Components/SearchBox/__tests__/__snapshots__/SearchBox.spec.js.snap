// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBox /> should render a search box 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-left: 110px;
}

.c0 button {
  color: #fff;
  background-color: #f65261;
  font-size: 15px;
  text-transform: uppercase;
  border: none;
  border-radius: 5%;
  outline: none;
  cursor: pointer;
  padding: 10px 20px;
  margin: 0;
  width: 160px;
}

.c0 input {
  width: 415px;
  height: 35px;
  color: #fff;
  margin-right: 15px;
  padding-left: 20px;
  background-color: #232323;
  border: none;
  border-radius: 5%;
  opacity: 0.5;
  outline: none;
}

.c0 input::-webkit-input-placeholder {
  font-size: 15px;
}

.c0 input::-moz-placeholder {
  font-size: 15px;
}

.c0 input:-ms-input-placeholder {
  font-size: 15px;
}

.c0 input::placeholder {
  font-size: 15px;
}

<div
  className="c0"
/>
`;
